reporting_app/__init__.py

```diff
@@ -440,14 +440,26 @@ def project_status_reports(prj_status):
             }
         )
 
+    # Last week project status page is a special case and needs to be rendered slightly differently.
+    if prj_status == 'lastweek':
```

Comment:
not too sure about overloading the project status with a time but since it might be a temporary change I would keep it as is.

Reply:
The project status is still being passed a string, I'm not overloading the function. I'm just handling this case differently as the page title needs to be generated differently. The filtering is also based on time rather than on the project_status.